src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs

```diff
@@ -248,7 +254,15 @@ public string CreateOffer(Bundle bundle)
             ValidateWebRTCState(WebRTCState.Negotiating);
 
             var bundle_ = bundle?.SafeBundleHandle ?? new SafeBundleHandle();
-            NativeWebRTC.CreateSDPOffer(Handle, bundle_, out string offer).ThrowIfFailed("Failed to create offer");
+
+            string offer = null;
+            var ret = NativeWebRTC.CreateSDPOffer(Handle, bundle_, out offer);
```

Comment:
I think the 30 timeout sec. in native API(it is latest one, but previous one was 10 sec.) enough to determine whether it is an error situation or not. What do you think about it?

Reply:
okay I'll remove retry logic.